docs/scheduler-http-api.md

```diff
@@ -128,6 +128,26 @@ TEARDOWN Response:
 HTTP/1.1 202 Accepted
 ```
 
+### SUPPRESS
+Sent by the scheduler when it wants to inform Mesos master to stop sending offers to the framework. When Mesos receives this request it will stop sending offers to framework but the tasks will continue running for `FrameworkInfo.failover_timeout`. Once the framework has work to do, it can send again a `REVIVE` request.
```

Comment:
How about:

Sent by the scheduler when it doesn't need offers for a given set of its roles. When Mesos master receives this request, it will stop sending offers for the given set of roles to the framework. As a special case, if `roles` is not specified, all subscribed roles of this framework are suppressed.

Note that master continues to send offers to other subscribed roles of this framework that are not suppressed. Also, status updates about tasks, executors and agents are not affected by this call. 

If the scheduler wishes to receive offers for the suppressed roles again (e.g., it needs to schedule new workloads), it can send `REVIVE` call.